### PR TITLE
show_bear: Show CAN_DETECT and CAN_FIX as well

### PR DESCRIPTION
**Pull request by sils1297**
Closes https://github.com/coala-analyzer/coala/issues/2441

### CODE REVIEW

File: coalib/results/Result.py [hunk 1]
```diff
@@ -35,7 +37,8 @@ def __init__(self,
                  severity: int=RESULT_SEVERITY.NORMAL,
                  additional_info: str="",
                  debug_msg="",
-                 diffs: (dict, None)=None):
+                 diffs: (dict, None)=None,
+                 confidence: int=100):
```
Comment (hypothesist) [inline coalib/results/Result.py]: Why not float?

**Comment by arafsheikh** [comment]
ack 979c40b

File: coalib/output/ConsoleInteraction.py [hunk 2]
```diff
@@ -697,6 +697,18 @@ def show_bear(bear,
                          metadata.optional_params,
                          "  ",
                          "No optional settings.")
+        show_enumeration(console_printer,
+                         "Can detect:",
+                         bear.can_detect,
+                         "  ",
+                         "This bear does not provide information about what "
```
Comment (hypothesist) [inline coalib/output/ConsoleInteraction.py]: Do we want to say this for each bear. A little spammy when I try to print 14-15 bears. IMO not printing the whole section is better.

File: coalib/results/Result.py (hunk 1 above)
Comment (sils1297) [inline coalib/results/Result.py]: float isn't precise, a percentage between 0 and 100 is definitely fine grained enough and simple to deal with without having to worry about precision issues with floats

File: coalib/output/ConsoleInteraction.py (hunk 2 above)
Comment (sils1297) [inline coalib/output/ConsoleInteraction.py]: hm I feel like this applies to others as well, make an issue maybe?

**Comment by hypothesist** [comment]
ack https://github.com/coala-analyzer/coala/pull/2445/commits/2c0bc424cd12ef019e3e5e59a81f8be465645ece

**Comment by sils1297** [comment]
@rultor merge then :)

**Pull request merged**